pyvista/plotting/plotting.py

```diff
@@ -2611,7 +2611,7 @@ def add_volume(
         scalars=None,
         clim=None,
         resolution=None,
-        opacity='linear',
+        opacity="linear",
```

Comment:
Just a drive-by comment for now: I see that a considerable part of the diff comes from string normalization like this. Since we explicitly skip these in `black`, see `skip-string-normalization` in https://github.com/pyvista/pyvista/blob/51fe884c50421e0889add8671a00abcc302f1a6e/pyproject.toml#L10-L14
I suspect this is due to some rogue tooling gone wild again. I would recommend not doing these tangential changes, because they make the PR harder to review, and they bloat the `git blame`. I can think of exceptions, e.g. if you notice a four-line exception message containing a mixture of single and double quotes.

Reply:
Thank you, yes. I will switch the quoting back.